tester.go

```diff
@@ -5,15 +5,20 @@ import (
 	"io"
 	"net/http"
 	"net/http/cookiejar"
-	"testing"
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
-// Tester represents the HTTP tester having testing.T.
+// TestingT is an interface wrapper around *testing.T.
+type TestingT interface {
```

Comment:
Why did you create a new `TestingT` interface without using `testing.TB`?

Reply:
Following the [stretchr/testify](https://github.com/stretchr/testify ) approach, I define only a minimal number of interfaces.

https://github.com/stretchr/testify/blob/1333b5d3bda8cf5aedcf3e1aaa95cac28aaab892/require/requirements.go#L3-L7